static/js/components/GroupShareSelect.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Select from "@material-ui/core/Select";
+import MenuItem from "@material-ui/core/MenuItem";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import Chip from "@material-ui/core/Chip";
+import Input from "@material-ui/core/Input";
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
```

Comment:
Could we define these in terms of the theme instead of raw pixels? You could use the useTheme hook and move these props into the component below.

Reply:
these were unnecesssary and I just removed them